src/Microsoft.Identity.Web/Resource/AadIssuerValidatorOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Options passed-in to create the AadIssuerValidator object.
+    /// </summary>
+    public class AadIssuerValidatorOptions
+    {
+        /// <summary>
+        /// Sets the name of the HttpClientFactory to use with the configuration manager.
+        /// Needed when setting up a proxy.
+        /// </summary>
+        public string? HttpClientFactoryName { get; set; } = null!;
```

Comment:
```suggestion
        /// Sets the name of the HttpClient to get from the IHttpClientFactory for use with the configuration manager.
        /// Needed when customizing client such as configuring a proxy.
        /// </summary>
        public string? HttpClientName { get; set; } = null!;
```

Reply:
@Tratcher: the thing I don't fully get is the proxy configuration is a static property of HttpClient? 
But I understand that for the default headers etc ... this is important.

Reply:
Yeah, you don't want the static, you want to set the instance property on SocketsHttpHandler when creating the HttpClient.
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.socketshttphandler.proxy?view=netcore-3.1#System_Net_Http_SocketsHttpHandler_Proxy

Reply:
@Tratcher i see. makes sense. will add to our documentation when we release.